lib/pundit/policy_finder.rb

```diff
@@ -23,9 +23,7 @@ def initialize(object)
     #   scope.resolve #=> <#ActiveRecord::Relation ...>
     #
     def scope
-      policy::Scope if policy
-    rescue NameError
-      nil
+      "#{policy}::Scope".safe_constantize
```

Comment:
Could you add an `if policy` here like it was before?

Reply:
Well, I guess it doesn't really matter when we use `.safe_constantize`...

Reply:
yep! `policy` will return either *something* or `nil` and 
```ruby
"#{nil}::Scope".safe_constantize"
=> nil
```